.gitattributes

```diff
@@ -7,3 +7,5 @@ tests/other/test_emsize.js -diff
 tests/* linguist-vendored
 third_party/* linguist-vendored
 system/ linguist-vendored
+
+AUTHORS merge=union
```

Comment:
this git change seems ok to me, what do you think @sbc100

Reply:
This is specifically to prevent merge conflicts like I have now.
That file shouldn't care about order of authors within, thus `merge=union` fits perfectly.

Reply:
Wow, I've not seen that before.  We should probably add that to CangeLog.md which always has merge conflicts near the top.

In this case I don't really know if it worth is, since the rate of new authors being added pretty low, and also it is cleaner if the new additions are always at the end.   But I don't feel strongly about this.

Reply:
I don't know about authors rate, but I had a merge conflict on that file _twice_ during this PR already :)

Reply:
Yeah, I agree this is a good idea!